src/api/hooks/useLogin.ts

```diff
@@ -25,7 +25,7 @@ const useLogin = () => {
             if (search) deleteQueryParams();
         }
     }, [client_info, client_account_params, client_object, search]);
-    return { is_logged_in };
+    return { is_logged_in, isAuthorized: isSuccess };
```

Comment:
For fixing the eslint error you need to change is_logged_in here too.
```suggestion
    return { isLoggedIn, isAuthorized: isSuccess };
```